index.html

```diff
@@ -13,8 +13,9 @@
       rel="stylesheet"
     />
     <link rel="stylesheet" href="style.css" />
-    <title>Document</title>
+    <title>My Portfolio Project</title>
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure you set a correct `title` for accessibility purposes of your web page on the internet :smile: 
`Hint: Raihan's portfolio would sound great than My portfolio`